src/Z38/SwissPayment/TransactionInformation/BankCreditTransfer.php

```diff
@@ -54,6 +54,8 @@ public function asDom(DOMDocument $doc, PaymentInformation $paymentInformation)
     {
         $root = $this->buildHeader($doc, $paymentInformation);
 
+        $root->appendChild($doc->createElement('ChrgBr', 'SHAR'));
```

Comment:
Vad är egentligen `ChrgBr` och `SHAR`?

Reply:
Specifies which party/parties will bear the charges associated with the processing of the payment transaction.

SHAR är enligt supporten hos SEB